targets/TARGET_STM/TARGET_STM32G4/todo.txt

```diff
@@ -0,0 +1 @@
+Still to be added: CAN, USBDEVICE, QSPI, CRC(?), TRNG
```

Comment:
Better to add these comments in the pull request ?

Reply:
Agreed

Reply:
2c5c6a75dcb3ce55a4ed0a5cce9a62c5b598e13d